projects/extension/sql/idempotent/012-vectorizer-int.sql

```diff
@@ -544,6 +617,70 @@ language plpgsql volatile security invoker
 set search_path to pg_catalog, pg_temp
 ;
 
+do $upgrade_block$
+declare
+    _vec record;
+    _new_version text := '0.8.1';
+begin
+    -- Find all vectorizers with version < 0.8.1
+    for _vec in (
+        select 
+            v.id,
+            v.source_schema,
+            v.source_table,
+            v.source_pk,
+            v.target_schema,
+            v.target_table,
+            v.trigger_name,
+            v.queue_schema,
+            v.queue_table,
+            v.config
+        from ai.vectorizer v
+        where string_to_array(regexp_replace((v.config->>'version'), '[-+].*$', ''), '.')::int[] < string_to_array(_new_version, '.')::int[]
+    )
+    loop
+        raise notice 'Upgrading trigger function for vectorizer ID %s from version %s', _vec.id, _vec.config->>'version';
+        
+        execute format(
+            'alter extension ai add function %I.%I()',
+            _vec.queue_schema, _vec.trigger_name
+        );
+
+        execute format(
+            'create or replace function %I.%I() returns trigger as $trigger_def$ %s $trigger_def$ language plpgsql volatile parallel safe security definer',
```

Comment:
we need to set the search_path explicitly here too

Reply:
whoops, i forgot to push this.